src/python/zquantum/qaoa/problems/generators.py

```diff
@@ -41,3 +41,36 @@ def get_random_hamiltonians_for_problem(
         hamiltonians.append(hamiltonian)
 
     return hamiltonians
+
+
+def get_random_hamiltonian(
```

Comment:
Also, the name of the function could be renamed to `get_random_ising_hamiltonian`.